parsec/core/gui/color.py

```diff
@@ -4,41 +4,41 @@
 import colorsys
 
 from binascii import crc32
+from typing import Tuple, cast
 
 
 class StringToColor:
-    def __init__(self, hue, lightness, saturation):
-        self.color = (hue, lightness, saturation)
+    def __init__(self, hue: float, lightness: float, saturation: float):
+        self.color: Tuple[float, float, float] = (hue, lightness, saturation)
```

Comment:
It can't infer type ?

Reply:
Yes, I removed this type hint as it it not required